modules/claims_api/spec/requests/v1/intent_to_file_request_spec.rb

```diff
@@ -229,11 +229,11 @@
             with_okta_user(scopes) do |auth_header|
               VCR.use_cassette('bgs/intent_to_file_web_service/insert_intent_to_file_500') do
                 data[:data][:attributes] = { type: 'pension' }
```

Comment:
The soap error handler raises the error right after the soap response is received and therefore does not create an intent_to_file in the DB. Is the the behavior we want?